vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/treegrid/it/TreeGridHugeTreeIT.java

```diff
@@ -79,6 +80,7 @@ public void collapsed_rows_invalidated_correctly() {
     }
 
     @Test
+    @Ignore
```

Comment:
> setViewportRange does not currently remove items from Cache and KeyMapper when they move out of the viewport. However, the previous version also only removed items from KeyMapper while keeping them in HierarchyMapper, so item instances actually continued to remain in memory. I feel this could be optimized separately if needed.
>
> https://github.com/vaadin/flow/pull/22019#issue-3313298808